compiler/optimizer/LocalOpts.cpp

```diff
@@ -8370,8 +8370,7 @@ TR_ColdBlockOutlining::reorderColdBlocks()
       // PLX: do not separate block with internal call
       // and the next block. The same for ASM block
 #if 0
-      if (currentBlock->isGenAsmBlock() &&
```

Comment:
Should this condition actually never be allowed to enter with your change since isGenAsmBlock should effectively be "false" ? i.e. there seems to be a logic change...is that intended ?